app/models/api/fake_data/grantee.rb

```diff
@@ -17,6 +17,7 @@ def data
       },
       links: {
         self: "https://example.com/TODO",
+        hses: "https://example.com/TODO",
         html: "https://example.com/TODO"
```

Comment:
[issue] we actually don't need to create a new URL -- if you check out the [API specs](https://github.com/OHS-Hosting-Infrastructure/complaint-tracker-api-docs/blob/main/hses/specifications/0.0.2/0.0.2.yml#L393-L394), the `html` URL is the one that should link to the hses grantee view. so we can just access the `html` link directly!

Reply:
Yep!  Let me know if you want a tour of the [OpenAPI specs](https://github.com/OHS-Hosting-Infrastructure/complaint-tracker-api-docs/blob/main/hses/specifications/0.0.2/0.0.2.yml) to talk through some of the grantee properties!

Reply:
👍🏾

Reply:
for prosperity: https://github.com/OHS-Hosting-Infrastructure/complaint-tracker-api-docs/blob/main/hses/specifications/0.0.2/0.0.2.yml#L393